core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -73,6 +73,28 @@ coreos:
 
         [Install]
         RequiredBy=format-etcd2-volume.service
+{{if and (.AmazonSsmAgent.Enabled) (ne .AmazonSsmAgent.DownloadUrl "")}}
+    - name: amazon-ssm-agent.service
+      command: start
+      enable: true
+      content: |
+        [Unit]
+        Description=amazon-ssm-agent
+        Requires=network-online.target
+        After=network-online.target
+
+        [Service]
+        Type=simple
+        WorkingDirectory=/
+        ExecStartPre=/opt/ssm/bin/install-ssm-agent.sh
+        ExecStart=/opt/ssm/bin/amazon-ssm-agent
+        KillMode=process
```

Comment:
I'm not an expert either 😉

Perhaps this should be `KillMode=control-group` if you'd want all the processes created for user commands run via the agent to be killed altogether?
I don't have a strong opinion here - am ok with either `process` or `control-group`.

@redbaron @andromedarabbit WDYT?